utility-classes.html

```diff
@@ -108,6 +108,22 @@
         flex-direction: row;
       }
 
+      /* === Flex grow === */
+      .v-flex-grow-0 {
+        flex-grow: 0;
+      }
+      .v-flex-grow {
+        flex-grow: 1;
+      }
+
+      /* === Flex shrink === */
+      .v-flex-shrink-0 {
+        flex-shrink: 0;
+      }
+      .v-flex-shrink {
+        flex-shrink: 1;
+      }
+
```

Comment:
I would expect to have `.v-flex-none` and `.v-flex-auto` available. I think those cover 80–90% of use cases.

Reply:
Added.